packages/themes/src/index.ts

```diff
@@ -0,0 +1 @@
+export { consoleLightTheme, consoleDarkTheme } from './themes'
```

Comment:
Why call that consoleThemes ? scw-ui should not be linked to shire right ?

Reply:
@chambo-e Right, maybe I can call it otherwise, like general theme or productTheme